collector/osv/src/client/mod.rs

```diff
@@ -68,9 +69,17 @@ impl OsvClient {
 
         Ok(response)
     }
+
+    pub async fn vulns(id: &str) -> Result<Vulnerability, anyhow::Error> {
+        let mut url = VULNS_URL.to_string();
+        url.push('/');
+        url.push_str(id);
+
```

Comment:
How about format!("{VULNS_URL}/{id}")

Reply:
I actually want to newtype around String to make constructing all client-y URLs easier. If'n it's okay, I'll do that subsequently.

Reply:
WFM